packages/merge/src/components/MergeSummary/MergeSummary.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {Typography} from 'tocco-ui'
+import {intlShape} from 'react-intl'
+import _groupBy from 'lodash/groupBy'
+
+const MergeSummary = ({mergeResponse, intl}) => {
+  const msg = id => intl.formatMessage({id})
+
+  const notCopiedRelations = Object.entries(_groupBy(mergeResponse.body.notCopiedRelations, e => {
+    return e.entityLabel
+  })).map(([entityLabel, list]) => `${entityLabel} (${list.length})`).join(', ')
+
+  const notDeletedEntities = Object.entries(_groupBy(mergeResponse.body.notDeletedEntities, e => {
+    return e.entityLabel
+  })).map(([entityLabel, list]) => `${entityLabel} (${list.length})`).join(', ')
+
+  return <>
+    <Typography.P>
+      {msg('client.merge.summary.success')}
```

Comment:
If possible (and here it is), use the react componenten rather than the function call.

<FormattedMessage id="client.merge.summary.success"/>